CMakeLists.txt

```diff
@@ -151,7 +151,9 @@ if(OCOS_ENABLE_PYTHON)
   if(NOT "${Python3_FIND_REGISTRY}" STREQUAL "NEVER")
     message(FATAL_ERROR "Python3_FIND_REGISTRY is not NEVER")
   endif()
+  message("-- Find Python3")
```

Comment:
Maybe we could delete one of message("-- Find Python3")

Reply:
done